client/Cargo.toml

```diff
@@ -21,6 +21,7 @@ iroha_config = { version = "=2.0.0-pre.1", path = "../config" }
 iroha_crypto = { version = "=2.0.0-pre.1", path = "../crypto" }
 iroha_data_model = { version = "=2.0.0-pre.1", path = "../data_model" }
 iroha_logger = { version = "=2.0.0-pre.1", path = "../logger"}
+iroha_telemetry = { version ="=2.0.0-pre.1", path = "../telemetry" }
```

Comment:
didn't we say this would be optional for the client, i.e. gated with `telemetry` feature?

Reply:
I figured that we rely on status, (which is a form of metric) in production code, so relying on telemetry is no longer optional. What I would make optional (in a separate PR) is the most expensive metrics. The reason I don't do it in this PR, is that I don't know which of the metrics will actually impact performance.